backend/api/Database/Models/MissionRun.cs

```diff
@@ -5,14 +5,14 @@
 #pragma warning disable CS8618
 namespace Api.Database.Models
 {
-    public class Mission
+    public class MissionRun
     {
         [Key]
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public string Id { get; set; }
 
         [MaxLength(200)]
-        public int? EchoMissionId { get; set; }
+        public string? MissionId { get; set; }
```

Comment:
```suggestion
        public MissionDefinition MissionDefinition { get; set; }
```

I don't think this should be optional, please correct me if I'm wrong. Also if we are pointing to the ID it makes more sense to point straight to the model.

Reply:
In addition, I think this field will be breaking for all old missions in the database.

Reply:
You could set the `MissionDefinition` as optional for now such that this does not break compatability with old missions and then create an issue which is to add a `MissionDefinition` for all old missions such that this one may be made required.

Reply:
I also think it would be nice to include MissionDefinition here. See this comment for why I have not yet implemented it: https://github.com/equinor/flotilla/pull/864#issuecomment-1621340083